includes/admin/class.llms.admin.builder.php

```diff
@@ -640,76 +643,184 @@ private static function process_detachments( $data ) {
 	/**
 	 * Delete/trash elements from heartbeat data
 	 *
-	 * @param    array $data  array of ids to trash/delete
-	 * @return   array
-	 * @since    3.16.0
-	 * @version  3.17.1
+	 * @since 3.16.0
+	 * @since 3.17.1 Unknown.
+	 * @since [version] Refactored method to reduce method complexity.
+	 *
+	 * @param array $data Array of ids to trash/delete.
+	 * @return array[]    Array of arrays containing information about the deleted items.
 	 */
 	private static function process_trash( $data ) {
 
 		$ret = array();
 
 		foreach ( $data['trash'] as $id ) {
+			array_push( $ret, self::process_trash_item( $id ) );
```

Comment:
I don't know, really. I used array_push() when I first wrote it and I never really thought about why... I'll adjust because I think `$array[] = 'el`;` is more readable anyway.